tests/test_run_steps_filter.py

```diff
@@ -1,7 +1,9 @@
 import pytest
 
+from universum import __main__
```

Comment:
Should we change the other test in this suite to running '__main__' instead of docker too? Maybe later in a separate PR?